.github/workflows/ci-rs.yml

```diff
@@ -61,8 +61,9 @@ jobs:
           RUSTDOCFLAGS: "-Dwarnings"
 
   benches:
+    name: Build benchmarks 🏋️
     needs: changes
-    if: ${{ needs.changes.outputs.rust == 'true' }} && github.event_name != 'merge_group'
+    if: ${{ needs.changes.outputs.rust == 'true' && github.event_name != 'merge_group' }}
```

Comment:
Can probably omit the `${{` and `}}`? But not important.

Reply:
I think they always enclose the full `if` expression in brackets:
https://docs.github.com/en/actions/learn-github-actions/expressions

Reply:
> I think they always enclose the full `if` expression in brackets: https://docs.github.com/en/actions/learn-github-actions/expressions

Seems for _most_ `if` expressions you can omit them: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif

But probably good practice to keep them as there are one or two exceptions.